style.css

```diff
@@ -0,0 +1,826 @@
+
+/* global */
+
+.hide {
+  display: none;
+}
+
+body {
+  max-width: 100%;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+main {
+  padding: 0 7px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body section {
+  margin-bottom: 2rem;
+}
+
+.flex {
+  display: flex;
+}
+
+.grid {
+  display: grid;
+}
+
+ul {
+  list-style: none;
+}
+
+.jc-space-between {
+  justify-content: space-between;
+}
+
+.align-items-center {
+  align-items: center;
+}
+
+.align-items-end {
+  align-items: flex-end;
+}
+
+.white-color {
+  color: white;
+}
+
+.gray-color {
+  color: rgb(145, 145, 145);
+}
+
+.orange-color {
+  color: #f42;
+}
+
+.big-section-container {
+  padding: 0 1rem;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.h3-headline {
+  font-size: 1.5rem;
+  font-weight: 700;
+  color: #f42;
+  margin-top: 2rem;
+  margin-bottom: 1rem;
+}
+
+.article-title {
+  font-size: 1.2rem;
+  font-weight: 700;
+  margin-bottom: 0.6rem;
+}
+
+.clear-fix {
+  clear: both;
+  display: table;
+  content: "";
+}
+
+/* links */
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.variety-news-related a {
+  display: block;
+  margin-left: 1rem;
+  padding: 0.5rem 0;
+}
+
+.latest-funding-container-options a {
+  display: block;
+  padding: 0.5rem;
+}
+
+.latest-news-deals .latest-news-deals-description a {
+  margin-right: 15px;
+}
+
+/* hover */
+
+.top-navbar a:hover {
+  color: white;
+}
+
+.on-hover-underline a:hover {
+  text-decoration: underline;
+}
+
+.on-hover-black a:hover {
+  color: black;
+}
+
+.on-hover-white a:hover {
+  color: white;
+}
+
+.on-hover-orange a:hover {
+  color: #f42;
+}
+
+/* li's */
+.footer-about-us li {
+  margin-right: 15px;
+}
+
+.variety-news-related li {
+  border-bottom: 1px solid #eee;
+  color: #333;
+}
+
+.latest-funding-headline-right li {
+  margin-top: 0.8rem;
+  font-size: 0.9rem;
+}
+
+.showcase .article-metadata li {
+  margin-right: 1rem;
+}
+
+/* images */
+.bottom-footer img {
+  height: 0.6rem;
+}
+
+header .brand img {
+  width: 65%;
+  position: absolute;
+  top: 15px;
+  left: 17px;
+}
+
+/* buttons */
+.footer-brand button {
+  color: white;
+  background-color: #333;
+  width: 41px;
+  height: 41px;
+  border-radius: 50%;
+  border: solid #333;
+  border-width: 2px;
+  font-size: 18px;
+  margin-right: 6px;
+}
+
+header .bottom-navbar-right button {
+  padding: 14px 16px;
+  background-color: inherit;
+  color: inherit;
+  border: transparent;
+  border-left: solid lightgray 1px;
+}
+
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<button>` element` and delete the left unnecessary rules


**Example:**
```CSS
/* before */
header .bottom-navbar-right button {

/* after */
.buttonelem-class {
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
Now resolved.